modules/rest-api/src/main/java/org/apache/ignite/internal/rest/api/recovery/GlobalPartitionStatesResponse.java

```diff
@@ -29,6 +30,7 @@
 @Schema(description = "Information about global partition states.")
 public class GlobalPartitionStatesResponse {
     @Schema
+    @JsonInclude
```

Comment:
Without JsonInclude, if states are empty, resulting body is `{}`, so deserialization fails ("states is null"). But didn't see any other usages of JsonInclude, so it seems there is a better way to fix it